encrypted-media/scripts/requestmediakeysystemaccess.js

```diff
@@ -0,0 +1,311 @@
+function runTest( config, qualifier ) {
+
+    var testname = testnamePrefix( qualifier, config.keysystem ) + ', requestMediaKeySystemAccess';
+
+    function expect_error(keySystem, configurations, expectedError, testname) {
+        promise_test(function(test) {
+            return navigator.requestMediaKeySystemAccess(keySystem, configurations).then(function(a) {
+                assert_unreached('Unexpected requestMediaKeySystemAccess() success.');
+            }, function(e) {
+                assert_equals(e.name, expectedError);
+            });
+        }, testname);
+    }
+
+    function assert_subset(actual, expected, path) {
```

Comment:
This function is comparing all string properties and length, right? Can we have here non-string properties? Should we check non-string properties?